metron-streaming/Metron-Common/src/main/java/org/apache/metron/bolt/BulkMessageWriterBolt.java

```diff
@@ -81,8 +81,7 @@ public void execute(Tuple tuple) {
       sensorMessageMap.put(sensorType, messageList);
     } else {
       try {
-        String esType = sensorType + "_doc";
-        bulkMessageWriter.write(esType, configurations, tupleList, messageList);
+        bulkMessageWriter.write(sensorType, configurations, tupleList, messageList);
```

Comment:
Yeah, I moved the _doc modification down into the ES writer rather than having it at the bolt level.  Should still work with the UI.